### ISSUE TITLE
No task runner configured when trying to build.

### ISSUE THREAD

**Issue by DavidRoys** [Microsoft/AL#45]
Created VM using http://aka.ms/navdeveloperpreview.

Logon to server and run Visual Studio Code from shortcut on desktop.

Open folder C:\Users\vmadmin\Documents\AL\Samples\HelloWorld

Press Ctr+Shift+B to build.

Get Info message: No task runner configured.

I see there is an option to configure a task runner but I have no idea which one I should select.

**Comment by DavidRoys** [comment]
OK I fixed this with some trial and error. 
In VS Code select the option to show extensions, then click the menu button (...) and choose Install from VSIX. 
Select: C:\DEMO\New Developer Experience\al-0.1.5449.vsix

No idea why the extension was not automatically added when the VM was initialised, but at least it's working now.

**Comment by freddydk** [comment]
David could you send me the installation status (c:\demo\status.txt) from this machine? This can maybe tell me why the .vsix installation wasn't done during initialization. Thanks.

**Comment by freddydk** [comment]
Thanks David
Did you run any of the scripts manually?
It seems like the initialization was complete at 8:12PM and the installation restarted at 1:16AM - and during the second run, the .vsix installation failed.
And it also re-started a 3rd time. Can you check whether there is a scheduled task in the Task Scheduler, which starts initialization every time the machine boots?
The Task should be removed upon successful installation of everything - but it seems like it hasn't happened.

**Comment by freddydk** [comment]
Just created a new instance - and indeed, the Task in the Task Scheduler is not deleted, but the folder c:\demo\install is deleted, so the task doesn't do anything.
Somehow that folder has not been deleted on your instance, which means that everytime your machine is rebooted, something goes wrong.
I would delete the instance and create a new one.

**Comment by DavidRoys** [comment]
I will try that. I notice that the machine will also restart without warning. I tried to turn off auto updates (which is the only change I made on the VM) but that didn't stop it.

**Comment by freddydk** [comment]
Yeah - I am afraid that the restart is at the end of the initialization script (which then starts on reboot), you have ended up in a never-ending reboot circle, sorry.

**Comment by DavidRoys** [comment]
OK. This time I gave the VM plenty of time to "finish cooking" before trying to connect and everyone seemed to be working fine when I eventually connected. I tend to stop the machine between sessions in order to preserve my Azure credit. I'm wondering if I had previously stopped the VM before it had finished running the setup. I'll post another comment if I get anything weird on this new one. Cheers.
